great_expectations/render/renderer/renderer.py

```diff
@@ -34,7 +33,6 @@ def inner_func(*args: P.args, **kwargs: P.kwargs):
     return wrapper
 
 
-@public_api
```

Comment:
We should limit the surface area of our public API - renderers are not particularly important to a user's understand of GX so let's hide it